test/cmd/get.sh

```diff
@@ -273,6 +278,10 @@ run_kubectl_sort_by_tests() {
   output_message=$(kubectl get pods --sort-by="{metadata.labels.name}")
   kube::test::if_sort_by_has_correct_order "${output_message}" "sorted-pod3:sorted-pod2:sorted-pod1:"
 
+  # if sorting, we should be able to use any field in our objects
+  output_message=$(kubectl get pods --sort-by="{spec.containers[0].name}")
+  kube::test::if_sort_by_has_correct_order "${output_message}" "sorted-pod2:sorted-pod1:sorted-pod3:"
+
```

Comment:
Can you add a test sorting by creation timestamp, just to ensure that other stuff are also working.